packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -888,7 +890,8 @@ export class LitContracts {
   public static getValidators = async (
     network: LIT_NETWORKS_KEYS,
     context?: LitContractContext | LitContractResolverContext,
-    rpcUrl?: string
+    rpcUrl?: string,
+    nodeProtocol?: typeof HTTP | typeof HTTPS | null
```

Comment:
Should we pick a proto to default to instead of allowing null?

Reply:
we already have a default here:
https://github.com/LIT-Protocol/js-sdk/pull/554/files#diff-2dcc2c94e7788b1bbaef5e6cf9158025a2580d238f807bd3cb923bbe355eb47bR958

`nodeProtocol` should be null by default, as we don't really want other to set except the internal devs